### DISCUSSION TITLE
Can only export as MP4

### DISCUSSION THREAD

**Discussion by AWeirdLizard** [mifi/lossless-cut#1597] [Q&A]
Hey all,

I've got a video file that is an mp4. I want to chop it up into a couple of gifs. But after trying various combinations of settings and tracks I couldn't get either gif or webm to export. They just say they fail and give me a file that can't be played. mp4 exports fine tho. .mov also works, mp3 does not. I haven't gone down the whole list yet at the time of writing this.

Any common things I might be doing wrong, or is there a bug that needs reporting? My original assumption was audio was a problem, but turning that off didn't help.

**Comment by mifi** [comment]
Hi. You cannot normally losslessly convert mp4 to gif because gif format probably doesn't support any of the codecs (h264,aac etc) that are normally inside mp4. see #372
